tools/js-sdk-release-tools/package.json

```diff
@@ -27,7 +27,7 @@
     "parse-ts-to-ast": "^0.1.1",
     "semver": "^7.3.5",
     "shelljs": "^0.8.4",
-    "simple-git": "^3.4.0",
+    "simple-git": "^3.5.0",
```

Comment:
@dw511214992 this change looks related to https://github.com/Azure/azure-sdk-tools/pull/3133. @chidozieononiwu any reason you didn't update the log file as part of this update?